Library/Formula/llvm.rb

```diff
@@ -129,9 +132,31 @@ def install
         # Let's use clang to build libcxx. It fails with gcc.
         ENV["CC"] = bin/"clang"
         ENV["CXX"] = bin/"clang++"
+        # in order to build libcxx on Linux you have to set libsupc++ and
+        # provide its includes
+        # Source: https://solarianprogrammer.com/2013/01/17/building-clang-libcpp-ubuntu-linux/
+        libcxx_args = ["-DLIBCXX_CXX_ABI=libsupc++"]
+        # To find which folders to include we iterate through GCC system
+        # directories looking for files libcxx needs (cxxabi.h and
+        # /bits/c++config.h)
+        libcxx_includes = `#{"
+          echo | g++ -Wp,-v -x c++ - -fsyntax-only 2>&1 | while read i;
```

Comment:
`echo | g++ -Wp,-v -x c++ - -fsyntax-only`
Better style is:
`g++ -Wp,-v -x c++ - -fsyntax-only </dev/null`